svg/modern/right_ptr.svg

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<svg xmlns="http://www.w3.org/2000/svg" width="256" height="256" viewBox="0 0 256 256" fill="none">
```

Comment:
This (for all cursors) likely needs hotspot adjustment in `build.toml`, also:

https://github.com/ful1e5/Bibata_Cursor/blob/f9704f65856d7675e27b7ce9b5d7833106f9771d/build.toml#L280-L281